sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/common/ImmutableDoubleArray.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.common;
+
+import java.util.Arrays;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class ImmutableDoubleArray {
+  private static final ImmutableDoubleArray EMPTY = new ImmutableDoubleArray(new double[0]);
+
+  /** Returns the empty array. */
+  public static ImmutableDoubleArray of() {
+    return EMPTY;
+  }
+
+  /** Returns an immutable array containing a single value. */
+  public static ImmutableDoubleArray of(double e0) {
```

Comment:
this we can probably remove, too, since a single-bucket histogram seems...not very useful

Reply:
yep, not very useful in practise, but I mainly use it in the unit tests. otherwise it's a bit pain to write lots of `ImmutableDoubleArray.of(new double[]{1})` instead of `ImmutableDoubleArray.of(1)`.